src/coreclr/vm/stubhelpers.cpp

```diff
@@ -528,10 +525,6 @@ FCIMPL2(void*, StubHelpers::GetDelegateTarget, DelegateObject *pThisUNSAFE, UINT
     // The lowest bit is used to distinguish between MD and target on 64-bit.
     target = (target << 1) | 1;
 
-    // On 64-bit we pass the real target to the stub-for-host through this out argument,
-    // see IL code gen in NDirectStubLinker::DoNDirect for details.
-    *ppStubArg = target;
-
 #elif defined(TARGET_ARM)
     // @ARMTODO: Nothing to do for ARM yet since we don't support the hosted path.
 #endif // HOST_64BIT, TARGET_ARM
```

Comment:
`pEntryPoint` is always NULL here. `if (pEntryPoint == NULL)` on the next line can be deleted.